CMakeLists.txt

```diff
@@ -188,6 +188,27 @@ if (SET_COMPILERS_AUTOMATICALLY)
   PROJECT(Albany)
 endif()
 
+include(CheckCSourceCompiles)
+set(CMAKE_REQUIRED_LIBRARIES ${${PROJECT_NAME}_EXTRA_LINK_FLAGS})
+set(CMAKE_REQUIRED_INCLUDES  ${Trilinos_INCLUDE_DIRS})
+check_c_source_compiles(
+  "
+  #include <exodusII_par.h>
+  int main()
+  {
+    return 0;
+  }
+  "
+  PARALLEL_EXODUS_SUPPORTED
```

Comment:
Is there a reason not to just use `ALBANY_PARALLEL_EXODUS` as output variable?

Reply:
Not really. I thought that  
`Performing Test PARALLEL_EXODUS_SUPPORTED - Success`
was more expressive but at the same time wanted to keep Albany way of naming variables. I can change that.

Reply:
No, it's fine. I was just curious if there was some weird techincality in the CheckFunctionExists macro maybe. This is fine.